packages/account/src/Containers/account.jsx

```diff
@@ -106,54 +106,61 @@ const Account = ({
     }
 
     const selected_route = getSelectedRoute({ routes: subroutes, pathname: location.pathname });
+    const getPageOverlay = () => {
```

Comment:
This should be made as a separate component. Otherwise it's gonna create dom nodes each time.
https://stackoverflow.com/questions/58507656/why-does-react-discard-the-entire-dom-subtree-and-recreate-it-from-scratch

Reply:
We can also set it to a variable.
```
let element;

if (condition) {
   element = <PageOverlay> .... 
} else {
   element = <VerticalTab> ...
}
```
and use it in render fn.

Reply:
i think declaring it outside of `Account` will also work